govcd/vm_test.go

```diff
@@ -1430,3 +1422,38 @@ func (vcd *TestVCD) Test_QueryVmList(check *C) {
 		check.Assert(foundVm, Equals, true)
 	}
 }
+
+// Test update of VM Capabilities
+func (vcd *TestVCD) Test_UpdateVmCapabilities(check *C) {
+	fmt.Printf("Running: %s\n", check.TestName())
+
+	vmName := "Test_UpdateVmCapabilities"
+	if vcd.skipVappTests {
+		check.Skip("Skipping test because vApp wasn't properly created")
+	}
+
+	vdc, _, vappTemplate, vapp, desiredNetConfig, err := vcd.createAngGetResourcesForVmCreation(check, vmName)
```

Comment:
There's a typo in this function name `Ang`->`And`

Reply:
renamed